src/lib/notificationHandlerWebsocket.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
```

Comment:
Wouldn't most of the actual logic be the same? I wonder if we actually would want multiple notification sending classes, versus something like having one that accepts some stream that could either be pure websockets, MQTT or something else